src/styles/blocks/calendar.scss

```
@@ -0,0 +1,40 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  gap: $gap;
+  max-width: $cell-size * 7 + $gap * 6;
+  margin: 80px auto;
+  line-height: $cell-size;
+  text-align: center;
```

```suggestion
  line-height: $cell-size;
  text-align: center;
```
line-height, text-align тут ни на что особо не влияет, контент выравнивается через justify-content и align-items в calendar__day

---

yes, you can remove it